.github/workflows/_test_template.yml

```diff
@@ -50,7 +53,21 @@ jobs:
         - name: Checkout repository
           uses: actions/checkout@v4
         - id: main
-          run: ${{ inputs.SCRIPT }}
+          name: Run main script
+          run: |
+            set +e 
+            (  
+              set -e
+
+              ${{ inputs.SCRIPT }}
+            ) 2> >(tee err.log)
+
+            EXIT_CODE=$?
+
+            echo "log=$(echo "$(tail -n 5 err.log)" | base64)" >> "$GITHUB_OUTPUT"
```

Comment:
Could we get 20 lines? Or 15?
Or at least 10? I think 5 may not give enough

Reply:
This was initially there since Slack has a maximum number of characters per message (3000). But I'm now tailing the last 2000 chars of our log record (leaving some headroom for the remaining part of the message), so that should give us a more informative alert